dali/pipeline/operator/op_spec.h

```diff
@@ -510,7 +484,10 @@ inline T OpSpec::GetArgumentImpl(
   } else {
     // Argument wasn't present locally, get the default from the associated schema
     const OpSchema& schema = GetSchema();
-    return static_cast<T>(schema.GetDefaultValueForOptionalArgument<S>(name));
+    DALI_ENFORCE(
+        schema.HasArgumentDefaultValue(name),
+        make_string("Argument \"", name, "\" has no default value and was not set."));
+    return static_cast<T>(schema.GetDefaultValueForArgument<S>(name));
```

Comment:
I think that the underlying call should take care of there not being a default value, shouldn't it?

Reply:
Done